src/plugins/text/render.js

```diff
@@ -14,6 +14,7 @@ const MARKS = {
     underline: 'u',
     superscript: 'sup',
     subscript: 'sub',
+    code: 'code',
```

Comment:
Plugins should only render nodes which they define, and plugins which define a node should also render it.

Reply:
Not sure about this. If we have all others marks in one place then why change it?

Reply:
Plugins are (supposed to be, by design) independent from each other. So if you disable plugin `Text`, `Code` should work without any change in behaviour, but as it is right now, inline code tags will stop rendering.